snowfakery/output_streams.py

```diff
@@ -375,16 +382,24 @@ def write_single_row(self, tablename: str, row: Dict) -> None:
         self.G.add_node(node_name)
 
     def close(self) -> None:
-        self.file.write(self.G.string())
+        self.stream.write(self.G.string())
+        super().close()
 
 
 class ImageOutputStream(GraphvizOutputStream):
-    def __init__(self, file, format="png"):
+    """Output an Image file in a graphviz supported format."""
+
+    mode = "wb"
+
+    def __init__(self, path, format):
+        self.path = path
         self.format = format
-        super().__init__(file)
+        super().__init__(path)
 
     def close(self) -> None:
-        self.G.draw(path=self.file, prog="dot", format=self.format)
+        if isinstance(self.path, Path):
+            self.path = str()
```

Comment:
Huh?

Reply:
Removed. It was both obsolete and incomplete.